api/src/main/java/org/apache/cloudstack/usage/UsageUnitTypes.java

```diff
@@ -0,0 +1,52 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.cloudstack.usage;
+
+public enum UsageUnitTypes {
+    ComputeMonth ("Compute-Month"),
+    IpMonth ("IP-Month"),
+    Gb ("GB"),
+    GbMonth ("GB-Month"),
+    PolicyMonth ("Policy-Month");
```

Comment:
should these be per month? e.g. `IP/Month` (or `IP*Month`)?

Reply:
@DaanHoogland, they follow the same logic as before, meaning the cost of 1 resource (1 compute, 1 IP, 1Gb)  per month. Should we change the nomenclature?

Note: the usage unit types are the same, I only turned them into enums.

Reply:
I think we can "improve" by using more scientific notation. If they are per month it should read `/month` in my not so humble opinion. Not going to cry block this for this reason.

Reply:
Reading further I think most of these are `*month` and not `/month`, as would make sense in billing (think of kWh).